### PR TITLE
docs(README): centered logos, adds slack badge, adds license badge

### PR DESCRIPTION
**Pull request by aayushbisen**
Signed-off-by: Aayush Bisen <aayush23101998@outlook.com>

Centered Cicero and Accord Project Logo, adding the Slack badge, adding the License badge

### Changes
- Centered logos of Cicero and Accord Project in README
- Adding the Slack Badge in README
- Adding the license badge in README

### Flags
- None

### Related Issues
- None

### CODE REVIEW

**Comment by irmerk** [comment]
Thanks for the PR! Could you reference the issue this is for?

**Comment by aayushbisen** [comment]
This PR solves the following issues:

- https://github.com/accordproject/techdocs/issues/78
- https://github.com/accordproject/techdocs/issues/79
- https://github.com/accordproject/techdocs/issues/82

File: README.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+</p>
+
 
 [![Build Status](https://travis-ci.org/accordproject/cicero.svg?branch=master)](https://travis-ci.org/accordproject/cicero)
 [![Coverage Status](https://coveralls.io/repos/github/accordproject/cicero/badge.svg?branch=master)](https://coveralls.io/github/accordproject/cicero?branch=master)
+[![GitHub license](https://img.shields.io/github/license/accordproject/ergo?color=bright-green)](./LICENSE)
 [![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
 [![npm version](https://badge.fury.io/js/%40accordproject%2Fcicero-cli.svg)](https://badge.fury.io/js/%40accordproject%2Fcicero-cli)
+[![join slack](https://img.shields.io/badge/Slack-Join%20Slack-blue)](https://accord-project-slack-signup.herokuapp.com/)
+
```
Comment (irmerk) [inline README.md]: Are you able to center these as well, to fit under `Cicero`'s logo?

Comment (aayushbisen) [reply README.md]: Done. Please review!

**Comment by mttrbrts** [comment]
Good job @aayushbisen! Thanks for the PR

**Pull request merged**